Packages/manifest.json

```diff
@@ -1,11 +1,18 @@
 {
   "dependencies": {
-    "com.unity.ads": "2.0.8",
-    "com.unity.analytics": "3.3.2",
+    "com.unity.2d.sprite": "1.0.0",
+    "com.unity.2d.tilemap": "1.0.0",
```

Comment:
uhhh whats with all the dependency changes

Reply:
no idea, could it be to do with upgrading the project version? I think develop is still 2019.3.7f1.
I remember you upgraded it but that was on the messages-ui branch?

Reply:
Either way, is it a big deal?

Reply:
lol not really, was just curious what happened